worldedit-core/src/main/java/com/fastasyncworldedit/core/command/brush/heightmap/AbstractDelegateHeightMap.java

```diff
@@ -1,4 +1,4 @@
-package com.fastasyncworldedit.core.object.brush.heightmap;
+package com.fastasyncworldedit.core.command.brush.heightmap;
```

Comment:
Not sure heightmap classes should be specifically under brush

Reply:
How about function.heightmap?

Reply:
`extension.processor.heightmap` perhaps